diff --git a/node_modules/slate/lib/slate.es.js b/node_modules/slate/lib/slate.es.js
index 1781173..da6f21a 100644
--- a/node_modules/slate/lib/slate.es.js
+++ b/node_modules/slate/lib/slate.es.js
@@ -13437,7 +13437,7 @@ var ElementInterface = function () {
           previousPath = _previous[1];

       if (closestBlock.hasDescendant(previousPath)) {
-        return previous.getMarksAtIndex(previousText.text.length);
+        return previousText.getMarksAtIndex(previousText.text.length);
       }

       return currentMarks;
